prototypes/ItemRepeaterPrototypes/ItemsRepeaterExperiments/AttachedBehaviors/SelectionModelPage.xaml.cs

```diff
@@ -0,0 +1,102 @@
+﻿using ItemsRepeaterExperiments.Common;
+using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace ItemsRepeaterExperiments.AttachedBehaviors
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class SelectionModelPage : Page
+    {
+        private List<Team> teams = DataSourceCreator<Team>.CreateRandomizedList(100);
+        private SelectionModel selectionModel = new SelectionModel();
+        public SelectionModelPage()
+        {
+            this.InitializeComponent();
+            selectionModel.Source = teams;
+            selectionModel.SelectionChanged += SelectionModel_SelectionChanged;
+            selectionModel.SingleSelect = true;
+        }
+
+
+        /// <summary>
+        /// Attach properties to virtualized items and update containers
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="args"></param>
+        private void ItemsRepeater_ElementPrepared(Microsoft.UI.Xaml.Controls.ItemsRepeater sender, Microsoft.UI.Xaml.Controls.ItemsRepeaterElementPreparedEventArgs args)
+        {
+            new PointerBehaviors(args.Element as Control).Click += Item_Click;
+            var selectionbehavior = new SelectionBehavior(args.Element as Control);
+
+            if(selectionModel.SelectedIndex != null && selectionModel.SelectedIndex.GetAt(0) == args.Index)
+            {
+                selectionbehavior.IsSelected = true;
+            }
+            else
+            {
+                selectionbehavior.IsSelected = false;
+            }
```

Comment:
This can be replaced with just
SelectionBehavior.SetIsSelected(args.Element as Control,
                selectionModel.SelectedIndex != null && selectionModel.SelectedIndex.GetAt(0) == args.Index);